src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -2464,15 +2288,6 @@ public void mySignal(String value) {
 
   @Test
   public void testSignal() {
-    // Test getTrace through replay by a local worker.
```

Comment:
This block is a no-op, it create a new Worker and WorkerFactory but doesn't start it.